Source/WebCore/dom/TreeScope.cpp

```diff
@@ -583,7 +583,8 @@ struct SVGResourcesMap {
 
     MemoryCompactRobinHoodHashMap<AtomString, WeakHashSet<SVGElement, WeakPtrImplWithEventTargetData>> pendingResources;
     MemoryCompactRobinHoodHashMap<AtomString, WeakHashSet<SVGElement, WeakPtrImplWithEventTargetData>> pendingResourcesForRemoval;
-    MemoryCompactRobinHoodHashMap<AtomString, LegacyRenderSVGResourceContainer*> resources;
+//    MemoryCompactRobinHoodHashMap<AtomString, RenderSVGResourceContainer*> resources;
```

Comment:
We don't land commented code.

Reply:
Yeah, that is a good rule :) Though this PR is first to get some feedback.